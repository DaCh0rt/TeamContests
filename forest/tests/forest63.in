146 340
-8810 -9915
-4424 5678
6962 968
6725 2436
8859 8573
2516 -2804
-2939 4320
-6632 -3319
-3167 4608
1508 4610
-6188 8996
-9256 -8366
9633 -8872
-4537 -8174
6775 -3709
4716 8470
1292 -1073
4917 9416
2171 430
3476 -3404
-1655 -8965
-8559 -3361
-5905 -7166
6365 9851
-2839 223
6851 7009
-1175 -4342
6002 6132
-4661 1802
3028 1789
-375 7621
1025 -9237
1076 -5215
-9408 -3936
6826 5922
-6629 3135
2515 3506
-9164 -8137
-3786 3194
-2544 8737
-9230 2182
4486 1613
-5105 8954
7011 -5069
4390 5309
-1008 1985
8263 2848
-2733 -4306
-1515 -1620
-4368 6577
6925 8069
279 1954
3278 3235
890 5476
8837 3050
-2493 -787
8800 8283
-1184 9430
-1836 4149
3659 4887
9876 9862
-7663 -6930
-8430 9179
-8725 -6149
9393 3135
6661 -4861
-3931 -898
6391 -5183
6967 7474
-438 2087
-7001 4345
7169 -724
-6834 3461
-9097 4222
-6809 -9959
8038 -7106
-8403 -3612
5972 -3054
-4036 7310
-2552 1511
8083 -8853
-4259 -3716
-4900 8546
-2139 -6372
-5306 1948
-4764 9491
617 -9937
-541 -4855
83 6601
-8274 6936
-4498 -5219
-1990 9926
4123 9622
2284 -767
5151 6775
1409 -5126
1463 -5496
3265 -3571
4215 2980
6265 -3156
-9854 -1759
2286 -4936
-8323 1051
-8116 3838
-3614 -3499
-719 9273
5676 -2490
-1521 -9533
-3764 5382
7980 4614
-4506 7984
6816 7403
8821 5567
-1558 8771
3543 4874
2733 -6147
909 -2657
7034 -2052
-5304 5666
7614 -8356
-8124 -9363
9153 9807
-7290 -5330
860 -3119
6210 6133
-9502 8660
-5814 2268
-7708 5560
-4482 5168
9109 -6702
-7006 -6163
7042 1315
2862 8818
5687 -7943
-8644 4975
-2900 3373
6916 2221
-3032 -7060
6257 354
4639 9543
-7010 4698
355 8786
-5200 9047
-2149 3136
-7586 -2098
9870 7106
