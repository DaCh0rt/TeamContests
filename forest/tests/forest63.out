77589.51 401525009.88
